packages/cactus-plugin-ledger-connector-polkadot/src/test/typescript/integration/api-surface.test.ts

```diff
@@ -0,0 +1,8 @@
+import test, { Test } from "tape-promise/tape";
```

Comment:
Tape tests must be ignored in `jest.config.js` and listed in `.taprc`.

Also, it would be nice to use jest instead of tape (ignore it if it's too much effort, we can migrate it some other time).

Reply:
migrated to jest